Copyboard/AppDelegate.swift

```diff
@@ -13,13 +13,16 @@ import MenuBarKit
 import KeyboardShortcuts
 
 // MARK: - AppDelegate
+
+@MainActor
```

Comment:
since we have a single main actor, you can remove all the other ones that are redundent.